src/app/projects/projects-list/projects-list.component.ts

```diff
@@ -81,4 +88,54 @@ export class ProjectsListComponent implements OnInit, OnDestroy {
   changePage(page: PaginationEvent) {
     this.projectService.changePage(page.currentPage);
   }
+
+  saveSearchResultsAsTsv() {
+    const columns = {
+      uuid: 'Unique Key',
+      date: 'Date added',
+      title: 'Project Title',
+      publications: 'Publications',
+      authors: 'Authors',
+      organs: 'Organs',
+      technologies: 'Technologies',
+      cellCount: 'Cell count',
+      enaAccessions: 'ENA',
+      arrayExpressAccessions: 'Arrayexpress',
+      geoAccessions: 'GEO',
+      egaStudiesAccessions: 'EGA',
+      dcpUrl: 'HCA Data Portal URL'
+    };
+    const tsvArray = this.projectsAsTsvArray(this.filteredProjects, columns);
+    const tsvString = tsvArray.join('\r\n');
+    const blob = new Blob([tsvString], {type: 'text/tab-separated-values' });
+    saveAs(blob, 'HcaCatalogueExport.tsv');
```

Comment:
How about a timestamp in the filename as well? Might be better for users if they want a snapshot

Reply:
filename as requested by gabs